common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -52,7 +52,7 @@ class SequenceFields(object):
             "Note, you must enable Entrance Exams for this course setting to take effect."
         ),
         default=False,
-        scope=Scope.content,
+        scope=Scope.settings,
```

Comment:
@cpennington @chrisndodge can you guys look into this change ? 
I am changes it scope , `is_entrance_exam` flag was not appearing in course export <b>.tar.gz</b> file when scope was 'Scope.content'.